src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/PublicationController.cs

```diff
@@ -33,5 +33,12 @@ public async Task<ActionResult<PublicationTitleViewModel>> GetPublicationTitle(s
             return await this.JsonContentResultAsync<PublicationTitleViewModel>(() =>
                 _fileStorageService.DownloadTextAsync(PublicContentPublicationPath(slug)), NotFound());
         }
+
+        [HttpGet("publication/{slug}/methodology")]
+        public async Task<ActionResult<PublicationMethodologyViewModel>> GetPublicationMethodology(string slug)
+        {
+            return await this.JsonContentResultAsync<PublicationMethodologyViewModel>(() =>
+                _fileStorageService.DownloadTextAsync(PublicContentPublicationPath(slug)), NotFound());
+        }
```

Comment:
Just a comment, not sure if this is a problem - It's going to return 404 if the publication is not found, not if there's no methodology which I believe is possible.

Reply:
I think that's okay.